wikiget/__init__.py

```diff
@@ -22,5 +22,6 @@
 # set some global constants
 BLOCKSIZE = 65536
 DEFAULT_SITE = 'commons.wikimedia.org'
+DEFAULT_PATH = '/'
```

Comment:
```suggestion
DEFAULT_PATH = '/w/'
```

Reply:
Yes, you are right. I forgot to test the default Wikimedia site. Thank you!